### PR TITLE
Iss 948

### PR DESCRIPTION
**Pull request by dmgcodevil**

### CODE REVIEW

**Comment by spencergibb** [comment]
I look forward to reviewing this Monday!

**Comment by dmgcodevil** [comment]
@spencergibb thanks in advance, I haven't finish it yet, but you can take a look at the [test] (https://github.com/dmgcodevil/Hystrix/blob/master/hystrix-contrib/hystrix-javanica/src/test/java/com/netflix/hystrix/contrib/javanica/test/common/observable/BasicObservableTest.java) for observable, because PL is very big. BTW I didn't implement collapser feature for observable command, but hopefully will add it soon.

**Comment by mattrjacobs** [comment]
@dmgcodevil Is this ready for review and merge?  If so, I'll take a look this weekend

**Comment by dmgcodevil** [comment]
@mattrjacobs yes, it's pretty much done except Collapser for observable but it will change within separate issue

**Comment by spencergibb** [comment]
@dmgcodevil :+1:

**Comment by mattrjacobs** [comment]
LGTM.  Thanks for all your work on this

File: hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/command/ObservableResult.java [hunk 1]
```diff
@@ -1,46 +0,0 @@
-/**
```
Comment (dmgcodevil) [inline hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/command/ObservableResult.java]: Remove ObservableResult https://github.com/Netflix/Hystrix/issues/948

**Pull request merged**